app/templates/briefs/clarification_question.html

```diff
@@ -53,16 +53,17 @@
     {% endwith %}
   {% endif %}
 
-  {% with
-    heading = "Ask a question about ‘{}’".format(brief.title),
-    smaller = true
-  %}
-    {% include 'toolkit/page-heading.html' %}
-  {% endwith %}
-
   <div class="grid-row">
     <div class="column-two-thirds">
 
+      {% with
+        heading = "Ask a question about ‘{}’".format(brief.title),
+        smaller = true
+      %}
+        {% include 'toolkit/page-heading.html' %}
+      {% endwith %}
+
+
       <div class="hint">
         <p>Your question will be published with the buyers’ response by {{ brief.clarificationQuestionsClosedAt | dateformat }}.</p>
```

Comment:
Not part of this PR, but it worries me that the text here isn't true.  The date is the last date a question can be asked - from discussions the other day it seems like the answer might not get posted until the day before the opportunity closes.

A proper fix might need to wait for content sign-off for this page (the prototype doesn't include this new feature yet:
http://dm-prototype.herokuapp.com/page/ask-question/template-brief?key=outcomes.1), but we could remove the misleading date now, just to force whoever adds a date back in to think about what the right date is.